README.md

```diff
@@ -122,6 +122,14 @@ helper function, with the following arities:
   be added to the created HTTP map if desired, and will simply show up in the
   interface as extra fields.
 
+If using a ring-compliant server, a special `make-ring-request-info` helper is
+available to make creating Sentry request information from a ring request map
+easy:
+
+```clojure
+(add-http-info! (make-ring-request-info ring-request))
```

Comment:
How about `(add-ring-request! ring-request)` as a composition of the above two?

Reply:
Done. 

Maybe there is a nicer way to compose those functions, but since it escaped me I went ahead with a "full" implementation (the multiple arities makes it more tricky. But I'm happy to learn a new trick again :) )